[Pseudo_gene_schematic.png](https://zh.wikipedia.org/wiki/File:Pseudo_gene_schematic.png "fig:Pseudo_gene_schematic.png")，隨後一個基因發生突變，成爲假基因。一個新的學說認爲，假基因是基因轉錄的RNA逆轉錄並整合到DNA上形成的\[1\]\[2\]\]\]

**假基因**（Pseudogenes，Pseudo-意爲「假」）是一類染色體上的[基因](../Page/基因.md "wikilink")片段。假基因的序列通常與對應的[基因](../Page/基因.md "wikilink")相似，但至少是喪失了一部分功能，如基因不能[表達或](https://zh.wikipedia.org/wiki/基因表达 "wikilink")[編碼的](../Page/遗传密码.md "wikilink")[蛋白質沒有功能](https://zh.wikipedia.org/wiki/蛋白質 "wikilink")\[3\]。

一般認爲，假基因最初是功能對生物生存並非必要的基因。隨着[突變的積累](https://zh.wikipedia.org/wiki/基因突變 "wikilink")，出現編碼區提前出現[終止密碼子](https://zh.wikipedia.org/wiki/終止密碼子 "wikilink")、等情況，逐漸變爲無功能的假基因。另外，（Copy-number variation, CNV）也可能產生假基因。在拷貝數變異中，1kb（千[鹼基對](https://zh.wikipedia.org/wiki/鹼基對 "wikilink")）以上的DNA片段會發生複製或刪除\[4\]。一部分假基因既沒有[內含子](https://zh.wikipedia.org/wiki/內含子 "wikilink")，也沒有[啓動子](https://zh.wikipedia.org/wiki/啓動子 "wikilink")（這種啓動子被認爲是通過mRNA的逆轉錄轉移到染色體上的，稱爲「加工」假基因（processed pseudogenes））\[5\]，但部分假基因仍然擁有一些與正常基因相同的特徵，比如擁有[CpG島等啓動子](https://zh.wikipedia.org/wiki/CpG島 "wikilink")、[RNA剪接](../Page/RNA剪接.md "wikilink")位點等。

假基因這一名詞是由雅克（Jacq）等人於1977年最早提出的\[6\]。長期以來生物學家們認爲假基因是沒有功能的[垃圾DNA](https://zh.wikipedia.org/wiki/垃圾DNA "wikilink")，惟近年來的研究還表明假基因和其他非編碼片段一樣，擁有調控基因表達的功能。假基因的調控作用對維持生物體的生理活動有着重要意義，一部分假基因在某些疾病的發展中也扮演着重要角色\[7\]。

在進化生物學研究中，這些因為演化而喪失功能的假基因，對他們進行序列分析意義則相對重大，一直是研究者獲知生物進化歷程的手段。假基因一般會擁有一些源基因的特徵。按照[進化論的觀點](https://zh.wikipedia.org/wiki/進化論 "wikilink")，兩個親緣關係較近的物種擁有同一祖先。對假基因進行序列比對、分析，即可驗證兩物種是否擁有同一祖先，並能計算出兩物種開始分離的時間（結果能精確到百萬年）。

## 特性

Pseudogenes are usually characterized by a combination of [homology](https://zh.wikipedia.org/wiki/homology_\(biology\) "wikilink") to a known gene and loss of some functionality. That is, although every pseudogene has a [DNA](https://zh.wikipedia.org/wiki/DNA "wikilink") sequence that is similar to some functional gene, they are usually unable to produce functional final protein products.\[8\] Pseudogenes are sometimes difficult to identify and characterize in genomes, because the two requirements of homology and loss of functionality are usually implied through sequence alignments rather than biologically proven.

1.  Homology is implied by sequence identity between the DNA sequences of the pseudogene and parent gene. After [aligning](https://zh.wikipedia.org/wiki/sequence_alignment "wikilink") the two sequences, the percentage of identical [base pairs](https://zh.wikipedia.org/wiki/base_pair "wikilink") is computed. A high sequence identity means that it is highly likely that these two sequences diverged from a common ancestral sequence (are homologous), and highly unlikely that these two sequences have evolved independently (see [Convergent evolution](https://zh.wikipedia.org/wiki/Convergent_evolution "wikilink")).
2.  Nonfunctionality can manifest itself in many ways. Normally, a gene must go through several steps to a fully functional protein: [Transcription](https://zh.wikipedia.org/wiki/Transcription_\(genetics\) "wikilink"), [pre-mRNA processing](https://zh.wikipedia.org/wiki/pre-mRNA_processing "wikilink"), [translation](https://zh.wikipedia.org/wiki/translation_\(biology\) "wikilink"), and [protein folding](https://zh.wikipedia.org/wiki/protein_folding "wikilink") are all required parts of this process. If any of these steps fails, then the sequence may be considered nonfunctional. In high-throughput pseudogene identification, the most commonly identified disablements are premature [stop codons](https://zh.wikipedia.org/wiki/stop_codon "wikilink") and [frameshifts](https://zh.wikipedia.org/wiki/frameshift_mutation "wikilink"), which almost universally prevent the translation of a functional protein product.

Pseudogenes for [RNA](https://zh.wikipedia.org/wiki/RNA "wikilink") genes are usually more difficult to discover as they do not need to be translated and thus do not have "reading frames".

Pseudogenes can complicate molecular genetic studies. For example, amplification of a gene by [PCR](https://zh.wikipedia.org/wiki/polymerase_chain_reaction "wikilink") may simultaneously amplify a pseudogene that shares similar sequences. This is known as PCR bias or amplification bias. Similarly, pseudogenes are sometimes annotated as genes in [genome](https://zh.wikipedia.org/wiki/genome "wikilink") sequences.

Processed pseudogenes often pose a problem for [gene prediction](https://zh.wikipedia.org/wiki/gene_prediction "wikilink") programs, often being misidentified as real genes or exons. It has been proposed that identification of processed pseudogenes can help improve the accuracy of gene prediction methods.\[9\]

Recently 140 human pseudogenes have been shown to be translated.\[10\] However, the function, if any, of the protein products is unknown.

## 类型及成因

根据不同的起源机制和特点，假基因可大致分为如下四类：

### Processed

[Pseudogene2jpg.jpg](https://zh.wikipedia.org/wiki/File:Pseudogene2jpg.jpg "fig:Pseudogene2jpg.jpg") Processed (or [retrotransposed](https://zh.wikipedia.org/wiki/transposable_element "wikilink")) pseudogenes. In higher [eukaryotes](https://zh.wikipedia.org/wiki/eukaryote "wikilink"), particularly [mammals](https://zh.wikipedia.org/wiki/mammal "wikilink"), retrotransposition is a fairly common event that has had a huge impact on the composition of the genome. For example, somewhere between 30–44% of the [human genome](https://zh.wikipedia.org/wiki/human_genome "wikilink") consists of repetitive elements such as [SINEs](https://zh.wikipedia.org/wiki/Retrotransposon#SINEs "wikilink") and [LINEs](https://zh.wikipedia.org/wiki/Retrotransposon#LINEs "wikilink") (see [retrotransposons](https://zh.wikipedia.org/wiki/retrotransposons "wikilink")).\[11\]\[12\] In the process of retrotransposition, a portion of the [mRNA](https://zh.wikipedia.org/wiki/messenger_RNA "wikilink") or [hnRNA](https://zh.wikipedia.org/wiki/hnRNA "wikilink") transcript of a gene is spontaneously [reverse transcribed](https://zh.wikipedia.org/wiki/reverse_transcription "wikilink") back into DNA and inserted into chromosomal DNA. Although retrotransposons usually create copies of themselves, it has been shown in an *in vitro* system that they can create retrotransposed copies of random genes, too.\[13\] Once these pseudogenes are inserted back into the genome, they usually contain a [poly-A tail](https://zh.wikipedia.org/wiki/polyadenylation "wikilink"), and usually have had their introns [spliced out](https://zh.wikipedia.org/wiki/splicing_\(genetics\) "wikilink"); these are both hallmark features of [cDNAs](https://zh.wikipedia.org/wiki/cDNA "wikilink"). However, because they are derived from an RNA product, processed pseudogenes also lack the upstream promoters of normal genes; thus, they are considered "dead on arrival", becoming non-functional pseudogenes immediately upon the retrotransposition event.\[14\] However, these insertions occasionally contribute exons to existing genes, usually via [alternatively spliced](https://zh.wikipedia.org/wiki/alternative_splicing "wikilink") transcripts.\[15\] A further characteristic of processed pseudogenes is common truncation of the 5' end relative to the parent sequence, which is a result of the relatively non-processive retrotransposition mechanism that creates processed pseudogenes.\[16\] Processed pseudogenes are continually being created in primates.\[17\] Human populations, for example, have distinct sets of processed pseudogenes across its individuals.\[18\]

### Non-processed

[Pseudogene3jpg.jpg](https://zh.wikipedia.org/wiki/File:Pseudogene3jpg.jpg "fig:Pseudogene3jpg.jpg") Non-processed (or duplicated) pseudogenes. [Gene duplication](https://zh.wikipedia.org/wiki/Gene_duplication "wikilink") is another common and important process in the evolution of genomes. A copy of a functional gene may arise as a result of a gene duplication event caused by [homologous recombination](https://zh.wikipedia.org/wiki/homologous_recombination "wikilink") at, for example, repetitive [sine](https://zh.wikipedia.org/wiki/short_interspersed_element "wikilink") sequences on misaligned chromosomes and subsequently acquire [mutations](https://zh.wikipedia.org/wiki/mutation "wikilink") that cause the copy to lose the original gene's function. Duplicated pseudogenes usually have all the same characteristics as genes, including an intact [exon](https://zh.wikipedia.org/wiki/exon "wikilink")-[intron](https://zh.wikipedia.org/wiki/intron "wikilink") structure and regulatory sequences. The loss of a duplicated gene's functionality usually has little effect on an organism's [fitness](https://zh.wikipedia.org/wiki/fitness_\(biology\) "wikilink"), since an intact functional copy still exists. According to some evolutionary models, shared duplicated pseudogenes indicate the evolutionary relatedness of humans and the other primates.\[19\] If pseudogenization is due to gene duplication, it usually occurs in the first few million years after the gene duplication, provided the gene has not been subjected to any [selection pressure](https://zh.wikipedia.org/wiki/selection_pressure "wikilink").\[20\] Gene duplication generates functional [redundancy](https://zh.wikipedia.org/wiki/gene_redundancy "wikilink") and it is not normally advantageous to carry two identical genes. Mutations that disrupt either the structure or the function of either of the two genes are not deleterious and will not be removed through the selection process. As a result, the gene that has been mutated gradually becomes a pseudogene and will be either unexpressed or functionless. This kind of evolutionary fate is shown by population [genetic modeling](https://zh.wikipedia.org/wiki/genetic_model "wikilink")\[21\]\[22\] and also by [genome analysis](https://zh.wikipedia.org/wiki/genome_analysis "wikilink").\[23\]\[24\] According to evolutionary context, these pseudogenes will either be deleted or become so distinct from the parental genes so that they will no longer be identifiable. Relatively young pseudogenes can be recognized due to their sequence similarity.\[25\]

### Unitary pseudogenes

[Pseudogene4jpg.jpg](https://zh.wikipedia.org/wiki/File:Pseudogene4jpg.jpg "fig:Pseudogene4jpg.jpg") Various mutations (such as [indels](https://zh.wikipedia.org/wiki/indel "wikilink") and [nonsense mutations](https://zh.wikipedia.org/wiki/nonsense_mutation "wikilink")) can prevent a gene from being normally [transcribed](https://zh.wikipedia.org/wiki/transcription_\(biology\) "wikilink") or [translated](https://zh.wikipedia.org/wiki/Translation_\(biology\) "wikilink"), and thus the gene may become less- or non-functional or "deactivated". These are the same mechanisms by which non-processed genes become pseudogenes, but the difference in this case is that the gene was not duplicated before pseudogenization. Normally, such a pseudogene would be unlikely to become fixed in a population, but various population effects, such as [genetic drift](https://zh.wikipedia.org/wiki/genetic_drift "wikilink"), a [population bottleneck](https://zh.wikipedia.org/wiki/population_bottleneck "wikilink"), or, in some cases, [natural selection](https://zh.wikipedia.org/wiki/natural_selection "wikilink"), can lead to fixation. The classic example of a unitary pseudogene is the gene that presumably coded the enzyme [L-gulono-γ-lactone oxidase](https://zh.wikipedia.org/wiki/L-gulonolactone_oxidase "wikilink") (GULO) in primates. In all mammals studied besides primates (except guinea pigs), GULO aids in the biosynthesis of [ascorbic acid](https://zh.wikipedia.org/wiki/ascorbic_acid "wikilink") (vitamin C), but it exists as a disabled gene (GULOP) in humans and other primates.\[26\]\[27\] Another more recent example of a disabled gene links the deactivation of the [caspase 12](https://zh.wikipedia.org/wiki/caspase_12 "wikilink") gene (through a [nonsense mutation](https://zh.wikipedia.org/wiki/nonsense_mutation "wikilink")) to positive selection in humans.\[28\]

It has been shown that processed pseudogenes accumulate mutations faster than non-processed pseudogenes.\[29\]

### Pseudo-pseudogenes

The rapid proliferation of [DNA sequencing](https://zh.wikipedia.org/wiki/DNA_sequencing "wikilink") technologies has led to the identification of many apparent pseudogenes using [gene prediction](https://zh.wikipedia.org/wiki/gene_prediction "wikilink") techniques. Pseudogenes are often identified by the appearance of a premature [stop codon](https://zh.wikipedia.org/wiki/stop_codon "wikilink") in a predicted mRNA sequence, which would, in theory, prevent synthesis ([translation](https://zh.wikipedia.org/wiki/translation_\(biology\) "wikilink")) of the normal [protein](https://zh.wikipedia.org/wiki/protein "wikilink") product of the original gene. There have been some reports of translational readthrough of such premature stop codons in mammals, as reviewed in the "[Translational readthrough](https://zh.wikipedia.org/wiki/stop_codon#Translational_readthrough "wikilink")" section of the stop codon article. As alluded to in the figure above, a small amount of the protein product of such readthrough may still be recognizable and function at some level. If so, the pseudogene can be subject to [natural selection](https://zh.wikipedia.org/wiki/natural_selection "wikilink"). That appears to have happened during the evolution of *[Drosophila](https://zh.wikipedia.org/wiki/Drosophila "wikilink")* [species](https://zh.wikipedia.org/wiki/species "wikilink"), as described next. [Drosophila_melanogaster_-_side_(aka).jpg](https://zh.wikipedia.org/wiki/File:Drosophila_melanogaster_-_side_\(aka\).jpg "fig:Drosophila_melanogaster_-_side_(aka).jpg") In 2016 it was reported that 4 predicted pseudogenes in multiple *Drosophila* species actually encode proteins with biologically important functions,\[30\] "suggesting that such 'pseudo-pseudogenes' could represent a widespread phenomenon". For example, the functional protein (an [olfactory receptor](https://zh.wikipedia.org/wiki/olfactory_receptor "wikilink")) is found only in [neurons](https://zh.wikipedia.org/wiki/neurons "wikilink"). This finding of tissue-specific biologically-functional genes that could have been dismissed as pseudogenes by *[in silico](https://zh.wikipedia.org/wiki/in_silico "wikilink")* analysis complicates the analysis of sequence data. As of 2012, it appeared that there are approximately 12,000–14,000 pseudogenes in the human genome,\[31\] almost comparable to the oft-cited approximate value of 20,000 genes in our genome. The current work may also help to explain why we are able to live with 20 to 100 *putative* [homozygous](https://zh.wikipedia.org/wiki/homozygous "wikilink") [loss of function](https://zh.wikipedia.org/wiki/loss_of_function "wikilink") mutations in our genomes.\[32\]

Through reanalysis of over 50 million peptides generated from the human [proteome](https://zh.wikipedia.org/wiki/proteome "wikilink") and separated by [mass spectrometry](https://zh.wikipedia.org/wiki/mass_spectrometry "wikilink"), it now (2016) appears that there are at least 19,262 human proteins produced from 16,271 genes or clusters of genes. From this analysis, 8 new protein coding genes that were previously considered pseudogenes were identified.\[33\]

## 假基因功能的例子

1.  The term "pseudo-pseudogene" was coined in the publication that investigated the gene in the chemosensory [ionotropic glutamate receptor](https://zh.wikipedia.org/wiki/ionotropic_glutamate_receptor "wikilink") Ir75a of *[Drosophila sechellia](https://zh.wikipedia.org/wiki/Drosophila_sechellia "wikilink")*, which bears a premature termination codon (PTC) and was thus classified as a pseudogene based on that *in silico* analysis. However, *in vivo* the *D. sechellia* Ir75a locus produces a functional receptor, owing to translational read-through of the PTC. Read-through is detected only in neurons and depends on the nucleotide sequence downstream of the PTC.\[34\]
2.  The *[Drosophila](https://zh.wikipedia.org/wiki/Drosophila "wikilink")* *jingwei* gene produces a functional [alcohol dehydrogenase](https://zh.wikipedia.org/wiki/alcohol_dehydrogenase "wikilink") enzyme *in vivo*.\[35\] However, previous *in silico* analysis classified it as a processed pseudogene.\[36\] The evolution of this gene has been discussed.\[37\]
3.  A human processed pseudogene of phosphoglycerate mutase was initially reported by interpretation of both *in silico* and experimental evidence.\[38\] That pseudogene was investigated more fully by another group, which found convincing evidence that it was a functional gene,\[39\] which is now named . The gene is expressed in the testes and [polymorphisms](https://zh.wikipedia.org/wiki/polymorphism_\(biology\) "wikilink") in that gene appear to account for about 5% of cases of male infertility.\[40\]
4.  siRNAs. Some endogenous [siRNAs](https://zh.wikipedia.org/wiki/siRNA "wikilink") appear to be derived from pseudogenes, and thus some pseudogenes play a role in regulating protein-coding transcripts, as reviewed.\[41\] One of the many examples is psiPPM1K. Processing of RNAs transcribed from psiPPM1K yield siRNAs that can act to suppress the most common type of liver cancer, [hepatocellular carcinoma](https://zh.wikipedia.org/wiki/hepatocellular_carcinoma "wikilink").\[42\] This and much other research has led to considerable excitement about the possibility of targeting pseudogenes with/as therapeutic agents\[43\]
5.  Some [piRNAs](https://zh.wikipedia.org/wiki/Piwi-interacting_RNA "wikilink") are derived from pseudogenes located in piRNA clusters.\[44\] Those piRNAs regulate genes via the piRNA pathway in mammalian testes and are crucial for limiting [transposable element](https://zh.wikipedia.org/wiki/transposable_element "wikilink") damage to the genome.\[45\][BrafDFPjpg.jpg](https://zh.wikipedia.org/wiki/File:BrafDFPjpg.jpg "fig:BrafDFPjpg.jpg")
6.  There are many reports of pseudogene transcripts acting as [microRNA](https://zh.wikipedia.org/wiki/microRNA "wikilink") decoys. Perhaps the earliest definitive example of such a pseudogene involved in cancer is the pseudogene of [BRAF](https://zh.wikipedia.org/wiki/BRAF_\(gene\) "wikilink"). The BRAF gene is a [proto-oncogene](https://zh.wikipedia.org/wiki/proto-oncogene "wikilink") that, when mutated, is associated with many cancers. Normally, the amount of BRAF protein is kept under control in cells through the action of miRNA. In normal situations, the amount of RNA from BRAF and the pseudogene BRAFP1 compete for miRNA, but the balance of the 2 RNAs is such that cells grow normally. However, when BRAFP1 RNA expression is increased (either experimentally or by natural mutations), less miRNA is available to control the expression of BRAF, and the increased amount of BRAF protein causes cancer.\[46\] This sort of **c**ompetition for regulatory elements by RNAs that are **e**ndogenous to the genome has given rise to the term [**ce**RNA](https://zh.wikipedia.org/wiki/ceRNA "wikilink").
7.  The [PTEN](https://zh.wikipedia.org/wiki/PTEN_\(gene\) "wikilink") gene is a known [tumor suppressor gene](https://zh.wikipedia.org/wiki/tumor_suppressor_gene "wikilink"). The PTEN pseudogene, PTENP1 is a processed pseudogene that is very similar in its genetic sequence to the wild-type gene. However, PTENP1 has a missense mutation which eliminates the [codon](https://zh.wikipedia.org/wiki/codon "wikilink") for the [initiating methionine](https://zh.wikipedia.org/wiki/eukaryotic_translation "wikilink") and thus prevents translation of the normal PTEN protein.\[47\] In spite of that, PTENP1 appears to play a role in [oncogenesis](https://zh.wikipedia.org/wiki/oncogenesis "wikilink"). The 3' [UTR](https://zh.wikipedia.org/wiki/untranslated_region "wikilink") of PTENP1 mRNA functions as a decoy of PTEN mRNA by targeting [micro RNAs](https://zh.wikipedia.org/wiki/micro_RNA "wikilink") due to its similarity to the PTEN gene, and overexpression of the 3' UTR resulted in an increase of PTEN protein level.\[48\] That is, overexpression of the PTENP1 3' UTR leads to increased regulation and suppression of cancerous tumors. The biology of this system is basically the inverse of the BRAF system described above.
8.  Pseudogenes can, over evolutionary time scales, participate in [gene conversion](https://zh.wikipedia.org/wiki/gene_conversion "wikilink") and other mutational events that may give rise to new or newly-functional genes. This has led to the concept, used in a major review from 2003, that ***pseudo***genes could be viewed as ***pot***ogenes: ***pot***ential genes for evolutionary diversification.\[49\]

## 细菌假基因

[细菌](../Page/细菌.md "wikilink")基因组中也存在假基因\[50\]。这些拥有假基因的细菌通常为[共生](../Page/共生.md "wikilink")或[细胞内寄生](https://zh.wikipedia.org/wiki/细胞内寄生 "wikilink")，因此它们不需要一些生活在外界复杂环境中的细菌所必须的基因。一个极端的例子是[麻风病](../Page/麻风病.md "wikilink")的病原体--[麻风杆菌](../Page/麻风杆菌.md "wikilink")（*Mycobacterium leprae*）的基因组，已报道有1,133个假基因约占其[转录组](../Page/转录组.md "wikilink")的50%\[51\]。

## 參見

  - [逆轉座子](https://zh.wikipedia.org/wiki/逆轉座子 "wikilink")
  - [RNA干擾](https://zh.wikipedia.org/wiki/RNA干擾 "wikilink")

## 參考

## 拓展閱讀

  -
  -
  -
## 外部連結

  - [Pseudogene interaction database, miRNA-pseudogene and protein-pseudogene interaction maps database](https://web.archive.org/web/20110222111721/http://starbase.sysu.edu.cn/)
  - [Yale University pseudogene database](http://pseudogene.org/)
  - [Hoppsigen database](http://pbil.univ-lyon1.fr/databases/hoppsigen.html) (homologous processed pseudogenes)

[Category:遺傳學](https://zh.wikipedia.org/wiki/Category:遺傳學 "wikilink")

1.
2.
3.
4.
5.
6.
7.
8.
9.
10.
11.
12.
13.
14.
15.
16.
17.
18.
19.
20.
21.
22.
23.
24.
25.
26.
27.
28.
29.
30.
31.
32.
33.
34.
35.
36.
37.
38.
39.
40.
41.
42.
43.
44.
45.
46.
47.
48.
49.
50.
51.